app/views/admin/creator_settings/new.html.erb

```diff
@@ -42,6 +52,11 @@
   <div class="crayons-field mt-6 align-left">
     <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
   </div>
+   <% if @help_url %>
+      <a class="admin-help-button crayons-btn crayons-btn--icon-rounded crayons-btn--s" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
```

Comment:
This "help" icon looks a little off to me--it's too oblong. I've messed around with the styles quite a bit, but it doesn't appear to change the shape as expected. Additionally, I've compared the styles of this "help" icon to the styles of the other "help" icons scattered throughout the app, but haven't noticed anything jarringly different about the two. My hunch is that maybe the styles are being overridden some where/how? I'm using the same styles here that we do on the other "help" icons scattered throughout the codebase. 🕵🏻‍♀️  
![Screen Shot 2021-11-08 at 12 44 58 PM](https://user-images.githubusercontent.com/32834804/140807553-f3314729-659e-4669-a24d-b1d487a77674.png)